packer/install-dependencies.sh

```diff
@@ -188,8 +188,16 @@ for KUBE_DNS_MASQ_VERSION in ${KUBE_DNS_MASQ_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
-PAUSE_VERSIONS="3.1"
-for PAUSE_VERSION in ${PAUSE_VERSIONS}; do
+MCR_PAUSE_VERSIONS="1.2.0"
+for PAUSE_VERSION in ${MCR_PAUSE_VERSIONS}; do
+    # Pull the arch independent MCR pause image which is built for Linux and Windows
+    CONTAINER_IMAGE="mcr.microsoft.com/k8s/core/pause:${PAUSE_VERSION}"
```

Comment:
if I'm reading this right, we are already pulling the mcr image below

Reply:
```
# Image 'mcr.microsoft.com/k8s/azurestack/core/pause-amd64' is the same as 'k8s.gcr.io/pause-amd64'
    # At the time, re-tagging and pushing to mcr hub seemed simpler than changing how `defaults-kubelet.go` sets `--pod-infra-container-image`
    for IMAGE_BASE in k8s.gcr.io mcr.microsoft.com/k8s/azurestack/core; do
```

Reply:
oh nevermind that one is in the Azure Stack MCR... the renaming to MCR/GCR is confusing though

Reply:
maybe we can move the Azure Stack one to the MCR block, and also we should figure out if/why Azure Stack needs its own image, maybe one MCR image is good enough for everything

Reply:
We are pulling the image from a different path. @jackfrancis requested that we keep the old images pulled for [backward compat](https://github.com/Azure/aks-engine/pull/2050#discussion_r333119287).

This work basically did what Azure Stack didn't want to do in the PR comments above.

Reply:
In the PR discussion above, @jadarsie reviewed the change to move AzStack over to `mcr.microsoft.com/k8s/core/pause`. At this point, _all_ aks-engine clusters should depend on `mcr.microsoft.com/k8s/core/pause`.